Monika After Story/game/script-compliments.rpy

```diff
@@ -852,3 +852,73 @@ label mas_compliment_spending_time_2:
             m 2lksdlb "Well, I'm glad that you're entertained..."
             m 2ekd "...but that's not {i}quite{/i} what I had in mind."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_sweet",
+            prompt="You're really sweet!",
+            unlocked=False,
+            conditional="store.mas_anni.pastThreeMonths() and mas_isMoniAff(higher=True)",
+            action=EV_ACT_UNLOCK,
+            aff_range=(mas_aff.AFFECTIONATE, None)
+        ),
+        code="CMP"
+    )
+
+label monika_sweet:
```

Comment:
`eventlabel` needs to match
```suggestion
label mas_compliment_sweet:
```

Reply:
I think this was resolved without committing the change @dreamscached :)